go.mod

```diff
@@ -3,29 +3,23 @@ module github.com/helm/chart-releaser
 go 1.15
 
 require (
-	github.com/Azure/go-autorest/autorest/azure/auth v0.5.3 // indirect
-	github.com/Azure/go-autorest/autorest/to v0.3.0 // indirect
-	github.com/Azure/go-autorest/autorest/validation v0.2.0 // indirect
 	github.com/Songmu/retry v0.1.0
 	github.com/golangci/golangci-lint v1.37.0
-	github.com/google/go-github/v30 v30.0.0
-	github.com/goreleaser/goreleaser v0.129.0
-	github.com/jmoiron/sqlx v1.2.1-0.20190826204134-d7d95172beb5 // indirect
-	github.com/magefile/mage v1.10.0
+	github.com/google/go-github/v33 v33.0.0
+	github.com/goreleaser/goreleaser v0.156.2
+	github.com/magefile/mage v1.11.0
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.1.3
-	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
+	golang.org/x/oauth2 v0.0.0-20210216194517-16ff1888fd2e
 	golang.org/x/tools v0.1.0
-	helm.sh/helm/v3 v3.4.0
-	rsc.io/letsencrypt v0.0.3 // indirect; indirect)
+	helm.sh/helm/v3 v3.5.2
 )
 
-exclude (
-	github.com/Azure/go-autorest v0.9.0
-	github.com/Azure/go-autorest v12.0.0+incompatible
+replace (
+	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
```

Comment:
Out of interest - why is this needed? It doesn´t rewrite?

Reply:
I had to add that. That's due to the Helm dependency.

See https://github.com/helm/helm/blob/master/go.mod#L51-L54

Reply:
Otherwise, you get this error:

```go: helm.sh/helm/v3@v3.5.2 requires
	github.com/deislabs/oras@v0.10.0 requires
	github.com/docker/distribution@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000```